onnx_chainer/functions/loss.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+from onnx import helper
+from onnx.numpy_helper import from_array
+
+from onnx_chainer.onnx_helper import gensym
+from onnx_chainer import onnx_helper
+
+
+def convert_SoftmaxCrossEntropy(
+        func, opset_version, input_names,
+        num_outputs, parameters):
+    # obtain input variable
+    x_var, t_var = func.get_retained_inputs()
```

Comment:
Thank you for noting important point. Unfortunately, it seems like there's no easy way to retrieve input variables (`x` and `t`) just from the interface of `Function`... https://github.com/chainer/chainer/blob/v5.2.0/chainer/functions/loss/softmax_cross_entropy.py#L165
Since this converter is an experimental feature, I may want to a version requirements for this feature to keep the implementation simple. I may put an explicit error message when non-latest version Chainer is used.

Reply:
Nice!